Doc/library/datetime.rst

```diff
@@ -742,6 +742,8 @@ Other constructors, all class methods:
    If *tz* is not ``None``, it must be an instance of a :class:`tzinfo` subclass, and the
    current date and time are converted to *tz*’s time zone.  In this case the
    result is equivalent to ``tz.fromutc(datetime.utcnow().replace(tzinfo=tz))``.
+   It is recommended to create aware datetime object for current UTC time by
```

Comment:
Am not a native english speaker but shouldn't this "It is recommended to create aware datetime object " be "It is recommended to create an aware datetime object for current" or "be "It is recommended to create aware datetime objects for current"

Reply:
Am not a native english speaker neither but I'd propose:

    The recommended way to create aware datetime object for the current time in UTC is ...